server/routes/auth/auth.js

```diff
@@ -32,18 +40,23 @@ const getUserDetails = async (id, displayName) => {
 router.get('/auth/callback',
   passport.authenticate('OAuth2', { failureRedirect: '/bad' }),
   async function(req, res) {
-    // Successful authentication
+    // Successful 
     const user = await getUserDetails(req.user, req.user); // On whiten we'll take the display take from ping
 
-    // Check Ramad access
-    const isRamad = await checkIsAdmin(req.user);
-    req.session.admin = isRamad;
+    const { favorite_ads } = user;
+    const favoriteAds = favorite_ads[0] === null? [] : favorite_ads;
+
+    const MAX_AGE = 60 * 1000 * 2;
 
+    /* todo detrmine isRamad acording to WITH_RAMAD_ACCESS from '.env'
+    giving priority to the .env variable, need to convert string to boolean to*/
+    
     res.cookie('user', JSON.stringify({
-      upn: user.upn,
-      displayName: user.display_name,
-      isRamad: true /*When done, isRamad equals isRamad. For quick developing... you can do: false/true */
-    }));
+        upn: user.upn,
+        name: user.name,
+        isRamad: true,//user.is_ramad,
+        favoriteAds
+    }), { maxAge: MAX_AGE });
```

Comment:
I don't think we need to set an expiry here as this cookie is por presential info only.  
the real authentication cookie is set in the session and it's done automaticlly with `passport`.  
When we talked earlier you said sonthing about the expiry - we need to change it in the server index file where we set the cookies options:
```js
app.use(
  session({
    secret: sessionSecret,
    cookie: { maxAge: 60000 },
    resave: false,
    saveUninitialized: false
  })
);
```

Reply:
Got it, but still we have a problem after the max_age is finished

Reply:
I'm checking that